ui/webui/test/README.rst

```diff
@@ -1,17 +1,46 @@
-Integration Tests of Anaconda WebUI
-===================================
+Anaconda Web UI tests
+=====================
 
-This directory contains automated integration tests for Anaconda WebUI, and the support files for them.
+The Web UI tests are based on cockpit's testing framework which in turn is based
+on python's unittest library. There are two kinds of Web UI tests, "integration"
+that test only one specific feature at a time and "end to end" that go through the
+entire installation process including reboot to the installed system. Both types
+use the same library of helper functions, to avoid duplicating code and to make
+the test development easier, but they are executed in different ways because of
+the higher requirements of the end to end tests.
 
+Test development
```

Comment:
This whole section is too much information for a readme file. Part of these can be better inserted as docstrings in the code inline. Specific class names and decorator names are just too much detail for a README.